### ISSUE TITLE
String literals seem to be completely removed on parse

### ISSUE THREAD

**Issue by marcopolo1124** [phenomnomnominal/tsquery#96]
for any script I have with string literals e.g `console.log("hello world")`, doing
```javascript
const tree = ast(`console.log("hello world")`)
console.log(print(query(tree, "StringLiteral")[0]))
```
will yield `console.log()`

**Comment by marcopolo1124** [comment]
Just to add, the node that the query has the text "hello world" but printing this node will yield undefined

**Comment by phenomnomnominal** [comment]
Oh nice one! 

Think I [fixed this](https://github.com/phenomnomnominal/tsquery/commit/618343a8b83108be3d58ed625bfdbe09afc82850) and it's published as 6.1.3 🎉. Let me know if that fixes stuff
